kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -0,0 +1,11 @@
+{% extends "kolibri/base.html" %}
+{% load learn_tags kolibri_tags %}
+{% block frontend_assets %}
+{{ block.super }}
+{% frontend_assets 'kolibri.plugins.learn.KolibriLearnFrontEnd' %}
+{% content_renderer_frontend_async %}
```

Comment:
The purpose of the separate tags is to give a plugin developer a way to target their code. If it is only ever used on the learn tab I see no reason to have it loaded into every page, even if the amount of extra HTML it creates is minimal.

If there does become a use case for inserting it into all pages, then it can use that tag. I don't really want to default to overloading the base.html with everything, because I think it increases the risk of event collisions (someone specifies an event API in one KolibriModule, and it happens to have a name collision with another - we now have a KolibriModule responding to events it was not designed for.

So I guess my answer is - you should have to make an effort to load Javascript into a particular template, because that means you actually want it there.